cmake/generateConfigFile.cmake

```diff
@@ -39,7 +39,6 @@ int main() {
 check_include_file_cxx( "unistd.h"      EXV_HAVE_UNISTD_H )
 check_include_file_cxx( "sys/mman.h"    EXV_HAVE_SYS_MMAN_H )
 if ( NOT MINGW AND NOT MSYS AND NOT MSVC )
-check_include_file_cxx( "regex.h"       EXV_HAVE_REGEX_H )
 endif()
```

Comment:
Can lines 41 and 42 be removed too? It looks like an empty if now.

Reply:
Yes, totally! Thanks for spotting it :blush: